src/test/java/com/google/sps/TextAnalyserTest.java

```diff
@@ -0,0 +1,132 @@
+package com.google.sps.data;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.junit.Test;
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.Set;
+
+@RunWith(JUnit4.class)
+public final class TextAnalyserTest {
+
+  @Test
+  public void analysePositiveSentiment() {
+    try {
+      TextAnalyser textAnalyser = new TextAnalyser("Today is a special day. It is your birthday, so enjoy it!");
+      assertTrue("Sentiment score is not positive.", textAnalyser.getSentimentScore() > 0.0F);
+    } catch (IOException e) {
```

Comment:
You don't need to catch exceptions (unless you are testing behavior in abnormal execution flow).

Just `@throws IOException`.

Because if you do not expect your test to fail, and the subject under test throws an exception, the test will fail anyways. So the try-catch here and below is redundant.